virtcontainers/persist/fs/fs.go

```diff
@@ -77,21 +75,21 @@ func Init() (persistapi.PersistDriver, error) {
 	}, nil
 }
 
-func (fs *FS) sandboxDir() (string, error) {
-	id := fs.sandboxState.SandboxContainer
-	if id == "" {
-		return "", fmt.Errorf("sandbox container id required")
-	}
-
-	return filepath.Join(RunStoragePath(), id), nil
+func (fs *FS) sandboxDir(sandboxID string) (string, error) {
+	return filepath.Join(RunStoragePath(), sandboxID), nil
 }
 
 // ToDisk sandboxState and containerState to disk
 func (fs *FS) ToDisk(ss persistapi.SandboxState, cs map[string]persistapi.ContainerState) (retErr error) {
+	id := ss.SandboxContainer
+	if id == "" {
+		return fmt.Errorf("sandbox container id required")
```

Comment:
Nit: You can use `errors.New()` as there are no formats (same comment below).

Reply:
I think "fmt.Errorf" has the same effect with "errors.New()" so we can choose either one. 😄